definitions/npm/puppeteer_v1.2.x/flow_v0.58.x-/puppeteer_v1.2.x.js

```diff
@@ -0,0 +1,1437 @@
+// @flow
```

Comment:
This seems like you pulled it from somewhere? If so, where?

Reply:
@AndrewSouthpaw Yes, this was based on the [definitely typed](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/puppeteer) typescript definition. I've converted it by hand to flow

Reply:
Bravo. Can you link to the original in the top of the file, for reference by future contributors?

Reply:
@AndrewSouthpaw sure thing